python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -108,7 +108,7 @@ traffic-policies
 {%                                 endif %}
 {%                                 set protocol_cli = protocol_cli ~ ' source port field-set ' ~ protocol.src_field %}
          {{ protocol_cli }}
-{%                             elif protocol.icmp_type is arista.avd.defined(protocol.protocol) | lower == 'icmp' %}
+{%                             elif protocol.icmp_type is arista.avd.defined and protocol.protocol | lower == 'icmp' %}
```

Comment:
Seems it was never generating the `protocol icmp type` config so after fixing this able see the config